src/Microsoft.Framework.Runtime/NuGet/Authoring/ManifestReader.cs

```diff
@@ -50,6 +48,7 @@ private static ManifestMetadata ReadMetadata(XElement xElement)
                 {
                     ReadMetadataValue(manifestMetadata, element, allElements);
                 }
+
```

Comment:
I'm sure we can. But I keep the change to barely minimal. This is part of the original code of reading manifest. I lean to not to update it since we're going to refactory NuGet anyway.